examples/box/box/app.py

```diff
@@ -72,6 +82,9 @@ def set_blue_color(self, widget):
     def set_green_color(self, widget):
         self.outer_box.style.background_color = GREEN
 
+    def set_transparent_color(self, widget):
+        self.outer_box.style.background_color = TRANSPARENT
```

Comment:
Am I missing something? This looks like a really good test of the behavior change... but then it isn't hooked up to anything?

Reply:
Yes, it is 😅 
Removed it.